resources/views/requests/show.blade.php

```diff
@@ -230,6 +230,7 @@ class="nav-link"  @click="switchTab('editdata')">
                               :edit="configurationComments.edit"
                               :remove="configurationComments.remove"
                               :adding="configurationComments.comments"
+                              :readonly="'{{$request->status}}' === 'COMPLETED'"
```

Comment:
try to use the javascript/vue variable `request` instead of blade variable, to have something like this:
`:readonly="request.status === 'COMPLETED'"`
to be reactive with request status update